camelot/admin/object_admin.py

```diff
@@ -35,6 +35,7 @@
 
 from ..core.item_model.list_proxy import ListModelProxy
 from ..core.qt import Qt
+from ..core.orm.entity import EntityMeta
```

Comment:
geen van deze aanpassingen mogen in de ObjectAdmin zitten, maar moeten eigenlijk in de EntityAdmin zitten, ik ga hieronder over hoe ik de individuele aanpassingen mogelijk zie